core/domain/classifier_services.py

```diff
@@ -84,7 +84,7 @@ def classify_string_classifier_rule(state, normalized_answer):
     best_matched_rule_spec_index = None
 
     sc = classifier_registry.Registry.get_classifier_by_algorithm_id(
-        feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput'])
+        feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']['classifier_id'])
```

Comment:
@anmolshkl . It doesn't really matter since we can just remove the training from there with this PR. I'm just letting it stay so that the pipeline doesn't break. Actually , with very slight changes, we can make it use the cached classifiers. What do we do?